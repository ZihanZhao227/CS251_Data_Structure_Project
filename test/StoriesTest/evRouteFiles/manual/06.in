40
0 0 0 CITY Start
1 46 22 RESTAURANT RESTAURANT_1
2 178 24 CITY CITY_2
3 7 121 CHARGING_STATION CHARGING_STATION_3
4 24 33 RESTAURANT RESTAURANT_4
5 153 170 REGULAR_WAYPOINT REGULAR_WAYPOINT_5
6 163 138 TOURIST_ATTRACTION TOURIST_ATTRACTION_6
7 55 76 REGULAR_WAYPOINT REGULAR_WAYPOINT_7
8 135 90 CITY CITY_8
9 139 90 TOURIST_ATTRACTION TOURIST_ATTRACTION_9
10 119 64 CITY CITY_10
11 84 195 CITY CITY_11
12 52 100 REGULAR_WAYPOINT REGULAR_WAYPOINT_12
13 99 142 CITY CITY_13
14 95 6 REGULAR_WAYPOINT REGULAR_WAYPOINT_14
15 55 71 CHARGING_STATION CHARGING_STATION_15
16 147 105 CHARGING_STATION CHARGING_STATION_16
17 77 194 CHARGING_STATION CHARGING_STATION_17
18 84 137 CHARGING_STATION CHARGING_STATION_18
19 137 74 REGULAR_WAYPOINT REGULAR_WAYPOINT_19
20 120 145 REGULAR_WAYPOINT REGULAR_WAYPOINT_20
21 198 187 TOURIST_ATTRACTION TOURIST_ATTRACTION_21
22 128 174 CHARGING_STATION CHARGING_STATION_22
23 92 24 TOURIST_ATTRACTION TOURIST_ATTRACTION_23
24 110 100 TOURIST_ATTRACTION TOURIST_ATTRACTION_24
25 93 164 REGULAR_WAYPOINT REGULAR_WAYPOINT_25
26 28 121 CHARGING_STATION CHARGING_STATION_26
27 113 107 CHARGING_STATION CHARGING_STATION_27
28 150 132 RESTAURANT RESTAURANT_28
29 44 99 RESTAURANT RESTAURANT_29
30 24 165 TOURIST_ATTRACTION TOURIST_ATTRACTION_30
31 197 4 RESTAURANT RESTAURANT_31
32 52 196 CITY CITY_32
33 62 114 CHARGING_STATION CHARGING_STATION_33
34 97 169 CHARGING_STATION CHARGING_STATION_34
35 130 146 CHARGING_STATION CHARGING_STATION_35
36 196 182 CITY CITY_36
37 122 34 RESTAURANT RESTAURANT_37
38 122 97 CHARGING_STATION CHARGING_STATION_38
39 200 200 CITY End
