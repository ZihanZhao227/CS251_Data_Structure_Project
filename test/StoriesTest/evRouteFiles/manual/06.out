0 39 70.000 -1.0
NO_PATH
